### PR TITLE
MikeGrillo2

### PR DESCRIPTION
**Pull request by Mvgrillo1357**
Updated code to account for comment in a def

### CODE REVIEW

**Review by srwight — APPROVED** [review]
Looks good. Removed one line of whitespace.

File: mikeGrillo.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jan  3 14:05:29 2020
+
+@author: michael
+"""
+
+def grillom(a:int) -> str:
```
Comment (kylestormcloud) [inline mikeGrillo.py:9]: Takes in an integer

**Review by Maraudier — APPROVED** [review]
I am Matt Davis and I approve this message

File: mikeGrillo.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    """
+    this is my function
+
+    """
```
Comment (kylestormcloud) [inline mikeGrillo.py:13]: Multiline comment idented. Good.

**Review by kylestormcloud — APPROVED** [review]

**Pull request merged**